Author: <seneca@localhost.localdomain>
Description: Change default directories
--- ledgersmb-1.2.25.orig/LedgerSMB/Sysconfig.pm
+++ ledgersmb-1.2.25/LedgerSMB/Sysconfig.pm
@@ -30,13 +30,13 @@
 );
 
 # if you have latex installed set to 1
-$latex = 1;
+$latex = 0;
 
 # spool directory for batch printing
-$spool = "spool";
+$spool = "/var/lib/ledgersmb-1.2/spool";
 
 # path to user configuration files
-$userspath = "users";
+$userspath = "/var/lib/ledgersmb-1.2/users";
 
 # templates base directory
 $templates = "templates";
@@ -48,7 +48,7 @@
 $backuppath = $tempdir;
 
 # member file
-$memberfile = "users/members";
+$memberfile = "/var/lib/ledgersmb-1.2/users/members";
 
 # location of sendmail
 $sendmail = "/usr/sbin/sendmail -t";
@@ -64,19 +64,18 @@
 $check_max_invoices = 5;
 
 # program to use for file compression
-$gzip = "gzip -S .gz";
+$gzip = "/bin/gzip -S .gz";
 
 # Path to the translation files
-$localepath = 'locale/po';
+$localepath = '/usr/share/ledgersmb-1.2/locale/po';
 
 # available printers
 %printer = (
-    Laser => 'lpr -Plaser',
-    Epson => 'lpr -PEpson',
+    lpr => 'lpr',
 );
 
 my %config;
-read_config( 'ledgersmb.conf' => %config ) or die;
+read_config( '/etc/ledgersmb-1.2/ledgersmb.conf' => %config ) or die;
 
 # We should clean this code up for 1.3 :-) Chris T.
 
