Author: <seneca@localhost.localdomain>
Description: Adds Makefiles
--- /dev/null
+++ ledgersmb-1.2.25/bin/Makefile
@@ -0,0 +1,13 @@
+PACKAGE = ledgersmb
+
+FILES = $(shell ls *.pl)
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(DESTDIR)/usr/share/ledgersmb/bin/custom
+	cp $(FILES) $(DESTDIR)/usr/share/ledgersmb/bin
+	find $(DESTDIR)/usr/share/ledgersmb/bin -type d -exec chmod 755 '{}' \;
+	find $(DESTDIR)/usr/share/ledgersmb/bin -type f -exec chmod 644 '{}' \;
--- /dev/null
+++ ledgersmb-1.2.25/contrib/Makefile
@@ -0,0 +1,9 @@
+PACKAGE = ledgersmb
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(DESTDIR)/usr/share/doc/ledgersmb/contrib
+	cp -r * $(DESTDIR)/usr/share/doc/ledgersmb/contrib
--- /dev/null
+++ ledgersmb-1.2.25/css/Makefile
@@ -0,0 +1,13 @@
+PACKAGE = ledgersmb
+
+FILES = $(shell ls ledger-smb*.css)
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(DESTDIR)/var/lib/ledgersmb/css
+	cp $(FILES) $(DESTDIR)/var/lib/ledgersmb/css
+	chmod 644 $(DESTDIR)/var/lib/ledgersmb/css/*
+	ln -sf /var/lib/ledgersmb/css $(DESTDIR)/usr/share/ledgersmb
--- /dev/null
+++ ledgersmb-1.2.25/doc/Makefile
@@ -0,0 +1,18 @@
+PACKAGE = ledgersmb
+
+FILES = $(shell ls | grep -v Makefile | grep -v COPYING | grep -v COPYRIGHT)
+SUBDIRS = API manual database samples
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(DESTDIR)/usr/share/ledgersmb/doc
+	cp LedgerSMB-manual.pdf $(DESTDIR)/usr/share/ledgersmb/doc
+	mkdir -p $(DESTDIR)/usr/share/doc/ledgersmb
+	cp -a $(FILES) $(DESTDIR)/usr/share/doc/ledgersmb
+	cp -a API $(DESTDIR)/usr/share/doc/ledgersmb
+	cp -a manual $(DESTDIR)/usr/share/doc/ledgersmb
+	cp -a database $(DESTDIR)/usr/share/doc/ledgersmb
+	cp -a samples $(DESTDIR)/usr/share/doc/ledgersmb
--- /dev/null
+++ ledgersmb-1.2.25/drivers/Makefile
@@ -0,0 +1,10 @@
+PACKAGE = ledgersmb
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(DESTDIR)/usr/share/ledgersmb/drivers
+	cp *.pl $(DESTDIR)/usr/share/ledgersmb/drivers
+	chmod 644 $(DESTDIR)/usr/share/ledgersmb/drivers/*
--- /dev/null
+++ ledgersmb-1.2.25/LedgerSMB/Makefile
@@ -0,0 +1,16 @@
+PACKAGE = ledgersmb
+
+FILES := $(shell ls *.pm)
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(DESTDIR)/usr/share/ledgersmb/LedgerSMB
+	cp -r $(FILES) $(DESTDIR)/usr/share/ledgersmb/LedgerSMB
+	cp -r Taxes $(DESTDIR)/usr/share/ledgersmb/LedgerSMB
+	cp -r Session $(DESTDIR)/usr/share/ledgersmb/LedgerSMB
+	cp -r CreditCard $(DESTDIR)/usr/share/ledgersmb/LedgerSMB
+	find $(DESTDIR)/usr/share/ledgersmb/LedgerSMB -type f -exec chmod 644 '{}' \;
+	find $(DESTDIR)/usr/share/ledgersmb/LedgerSMB -type d -exec chmod 755 '{}' \;
--- /dev/null
+++ ledgersmb-1.2.25/locale/Makefile
@@ -0,0 +1,21 @@
+PACKAGE = ledgersmb
+
+LOCALES = $(shell ls po/*)
+
+localedir = /usr/share/ledgersmb/locale
+dlocaledir = $(DESTDIR)$(localedir)
+srctopdir := $(shell pwd)/..
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(dlocaledir)/po
+	mkdir -p $(dlocaledir)/html
+	cp po/* $(dlocaledir)/po
+	cp html/*.html $(dlocaledir)/html
+	cp LedgerSMB.pot $(dlocaledir)
+	cp *.html $(dlocaledir)
+	find $(dlocaledir) -type d -exec chmod 755 '{}' \;
+	find $(dlocaledir) -type f -exec chmod 644 '{}' \;
--- /dev/null
+++ ledgersmb-1.2.25/Makefile
@@ -0,0 +1,36 @@
+PACKAGE = ledgersmb
+
+SUBDIRS = bin doc drivers LedgerSMB locale sql templates css contrib utils
+
+LIB_FILES = favicon.ico menu.ini \
+	    ledgersmb.conf.default VERSION index.html \
+	    ledger-smb.png ledger-smb.gif ledger-smb_small.png \
+	    $(shell ls *.pl | grep -v -E '(pos\.conf)\.pl')
+
+CFG_FILES = ledgersmb.conf ledgersmb-httpd.conf pos.conf.pl
+
+all:
+
+install:
+	mkdir -p $(DESTDIR)/usr/share/ledgersmb
+	cp $(LIB_FILES) $(DESTDIR)/usr/share/ledgersmb
+
+	set -e ; for subdir in $(SUBDIRS) ; do \
+	  $(MAKE) -C $$subdir $@ ; \
+	done
+
+	mkdir -p $(DESTDIR)/etc/ledgersmb
+	cp $(CFG_FILES) $(DESTDIR)/etc/ledgersmb
+	cd $(DESTDIR)/usr/share/ledgersmb ; \
+	ln -sf /etc/ledgersmb/pos.conf.pl ; \
+	ln -sf /etc/ledgersmb/images
+	mkdir -p $(DESTDIR)/var/lib/ledgersmb/spool
+	chown www-data:www-data $(DESTDIR)/var/lib/ledgersmb/spool
+
+	mkdir -p $(DESTDIR)/etc/ledgersmb/images
+	mkdir -p $(DESTDIR)/etc/ledgersmb/images/demo
+	cp $(DESTDIR)/etc/ledgersmb/templates/demo/*.[^ht]* $(DESTDIR)/etc/ledgersmb/images
+	cp $(DESTDIR)/etc/ledgersmb/templates/demo/*.[^ht]* $(DESTDIR)/etc/ledgersmb/images/demo
+	chown -R www-data:www-data $(DESTDIR)/etc/ledgersmb/templates
+
+clean:
--- /dev/null
+++ ledgersmb-1.2.25/sql/Makefile
@@ -0,0 +1,14 @@
+PACKAGE = ledgersmb
+
+FILES = *.sql
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(DESTDIR)/usr/share/ledgersmb/sql/legacy
+	cp $(FILES) $(DESTDIR)/usr/share/ledgersmb/sql
+	cp -r legacy $(DESTDIR)/usr/share/ledgersmb/sql
+	find $(DESTDIR)/usr/share/ledgersmb/sql -type f -exec chmod 644 '{}' \;
+	find $(DESTDIR)/usr/share/ledgersmb/sql -type d -exec chmod 755 '{}' \;
--- /dev/null
+++ ledgersmb-1.2.25/templates/Makefile
@@ -0,0 +1,16 @@
+PACKAGE = ledgersmb
+
+FILES = *.html *.tex *.txt
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(DESTDIR)/etc/ledgersmb/templates
+	cp -a $(FILES) $(DESTDIR)/etc/ledgersmb/templates
+	cp -a demo $(DESTDIR)/etc/ledgersmb/templates
+	chown -R www-data:www-data $(DESTDIR)/etc/ledgersmb/templates
+	ln -sf /etc/ledgersmb/templates $(DESTDIR)/usr/share/ledgersmb/
+	find $(DESTDIR)/etc/ledgersmb/templates -type d -exec chmod 755 '{}' \;
+	find $(DESTDIR)/etc/ledgersmb/templates -type f -exec chmod 644 '{}' \;
--- /dev/null
+++ ledgersmb-1.2.25/utils/Makefile
@@ -0,0 +1,9 @@
+PACKAGE = ledgersmb
+
+all:
+
+clean:
+
+install:
+	mkdir -p $(DESTDIR)/usr/share/doc/ledgersmb/utils
+	cp -r * $(DESTDIR)/usr/share/doc/ledgersmb/utils
