Author:  Robert James Clay <jame@rocasa.us>, 
Description: Changing paths in Apache config
--- a/ledgersmb-httpd.conf
+++ b/ledgersmb-httpd.conf
@@ -1,7 +1,7 @@
 
-Alias /ledgersmb WORKING_DIR
+Alias /ledgersmb-1.2 /usr/share/ledgersmb-1.2
 
-<Directory WORKING_DIR>
+<Directory /usr/share/ledgersmb-1.2>
   AllowOverride All
   AddHandler cgi-script .pl
   Options ExecCGI Includes FollowSymlinks
@@ -13,22 +13,22 @@
   </Files>
 </Directory>
 
-<Directory WORKING_DIR/users>
+<Directory /usr/share/ledgersmb-1.2/users>
   Order Deny,Allow
   Deny from All
 </Directory>
-  
-<Directory WORKING_DIR/bin>
+
+<Directory /usr/share/ledgersmb-1.2/bin>
   Order Deny,Allow
   Deny from All
 </Directory>
-  
-<Directory WORKING_DIR/utils>
+
+<Directory /usr/share/ledgersmb-1.2/utils>
   Order Deny,Allow
   Deny from All
 </Directory>
-  
-<Directory WORKING_DIR/LedgerSMB>
+
+<Directory /usr/share/ledgersmb-1.2/ledgersmb>
   Order Deny,Allow
   Deny from All
 </Directory>
