Origin: upstream, http://ledger-smb.svn.sourceforge.net/ledger-smb/?rev=5042&view=rev
Description: Fix for fs_cssdir directory transversal error.
--- a/LedgerSMB/AM.pm
+++ b/LedgerSMB/AM.pm
@@ -1501,6 +1501,11 @@
 
     my @allowedsuff = qw(css tex txt html xml);
     my $test = $form->{file};
+    $test =~ s|^$LedgerSMB::Sysconfig::fs_cssdir||;
+    if ($LedgerSMB::Sysconfig::fs_cssdir
+           and $LedgerSMB::Sysconfig::fs_cssdir !~ m|/$|){
+         $test =~ s|^/||;
+    }
     if ($LedgerSMB::Sysconfig::templates =~ /^(.:)*?\//){
         $test =~ s#^$LedgerSMB::Sysconfig::templates/?\\?##;
     }
