Author: Robert James Clay <jame@rocasa.us>
Description: Change default directories
Last-Update: 2014-08-13
--- a/LedgerSMB/Sysconfig.pm
+++ b/LedgerSMB/Sysconfig.pm
@@ -20,8 +20,8 @@
 our $auth = 'DB';
 our $logging = 0;      # No logging on by default
 our $images = getcwd() . '/images'; 
-our $cssdir = 'css/';
-our $fs_cssdir = 'css/';
+our $cssdir = '/lsmbcss/';
+our $fs_cssdir = '/var/lib/ledgersmb/css/';
 
 our $force_username_case = undef; # don't force case
 
@@ -58,20 +58,20 @@
 our $cookie_name = "LedgerSMB-1.3";
 
 # spool directory for batch printing
-our $spool = "spool";
+our $spool = "/var/lib/ledgersmb/spool";
 
 our $cache_templates = 0;
 # path to user configuration files
-our $userspath = "users";
+our $userspath = "/var/lib/ledgersmb/users";
 
 # templates base directory
-our $templates = "templates";
+our $templates = "/var/lib/ledgersmb/templates";
 
 # Temporary files stored at"
 our $tempdir = ( $ENV{TEMP} || '/tmp' );
 
 # member file
-our $memberfile = "users/members";
+our $memberfile = "/var/lib/ledgersmb/users/members";
 
 # location of sendmail
 our $sendmail = "/usr/sbin/sendmail -t";
@@ -93,7 +93,7 @@
 our $gzip = "gzip -S .gz";
 
 # Path to the translation files
-our $localepath = 'locale/po';
+our $localepath = '/usr/share/ledgersmb/locale/po';
 
 our $no_db_str = 'database';
 our $log_level = 'ERROR';
--- a/ledgersmb.conf.default
+++ b/ledgersmb.conf.default
@@ -6,11 +6,11 @@
 # This is the logical CSS directory.  I.e. it is what comes before the 
 # ledgersmb.css in the url.  Example might be /my_css_dir/ or 
 # http://localhost/other_css_dir/
-cssdir = css/
+cssdir = /lsmbcss/
 
 # THis is the location on the file system where the css files are, for editing 
 # and selection.  An example might be /var/www/ledgersmb_css/
-fs_cssdir = css/
+fs_cssdir = /var/lib/ledgersmb/css/
 
 # If set to a true value this caches templates.  Typically it will be set to 0 
 # to disable or 1 to enable.
@@ -60,16 +60,16 @@
 
 [paths]
 # spool directory for batch printing
-spool      = spool
+spool      = /var/lib/ledgersmb/spool
 # path to user configuration files
-userspath  = users
+userspath  = /var/lib/ledgersmb/users
 # templates base directory
-templates  = templates
+templates  = /var/lib/ledgersmb/templates
 # images base directory
-images  = images
+images  = /usr/share/ledgersmb/images
 # member file
-memberfile = users/members
-localepath = locale/po
+memberfile = /var/lib/ledgersmb/users/members
+localepath = /usr/share/ledgersmb/locale/po
 
 [programs]
 # program to use for file compression
@@ -97,7 +97,7 @@
 db_namespace = public
 #contrib_dir PG_CONTRIB_DIR 
 #with postgresql-9.1 this stuff is in /usr/share/postgresql/9.1/extension and is injected in database with create extension pg_trgm,btree_gist...
-contrib_dir = /usr/share/pgsql/contrib/
+contrib_dir = /usr/share/postgresql/9.1/extension
 # sslmode can be require, allow, prefer, or disable.  Defaults to prefer.
 sslmode = prefer
 
